dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -728,7 +729,7 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink, public virtual
   EndpointSecurityAttributesMap endpoint_security_attributes_;
 #endif
 
-  void accumulate_addresses(const RepoId& local, const RepoId& remote, AddrSet& addresses) const;
+  void accumulate_addresses(const RepoId& local, const RepoId& remote, AddrSet& addresses, bool narrow = false) const;
```

Comment:
Keeping my theme, I would change `narrow` to `prefer_unicast` or `prefer_multicast`.